probatus/stat_tests/distribution_statistics.py

```diff
@@ -223,13 +226,16 @@ def __init__(self, statistical_tests="all", binning_strategies="default", bin_co
         Initializes the class.
 
         Args:
-            statistical_tests (str, optional): Statistical tests to apply, either list of tests names, or 'all'.
-                Statistical methods implemented:
+            statistical_tests (str or list of str, optional): Test or list of tests to apply.
+            Set to 'all' to apply all the available test. Available tests:
```

Comment:
I think here you have broken the table a bit:
![image](https://user-images.githubusercontent.com/40659437/129701449-75cc68ff-edb5-4fe3-9085-f5c6865fd518.png)

Reply:
Good catch! Indentation fixed now.
![image](https://user-images.githubusercontent.com/16015401/129714722-a072fb51-caf5-455c-a587-dcf1f953671b.png)